ros2component_test_fixtures/src/talker_component.cpp

```diff
@@ -0,0 +1,56 @@
+// Copyright 2019 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "ros2component_test_fixtures/talker_component.hpp"
+
+#include <chrono>
+#include <iostream>
+#include <memory>
+#include <utility>
+
+#include "rclcpp/rclcpp.hpp"
+#include "std_msgs/msg/string.hpp"
+
+using namespace std::chrono_literals;
+
+namespace ros2component_test_fixtures
+{
+
+// Create a Talker "component" that subclasses the generic rclcpp::Node base class.
+// Components get built into shared libraries and as such do not write their own main functions.
+// The process using the component's shared library will instantiate the class as a ROS node.
+Talker::Talker(const rclcpp::NodeOptions & options)
+: Node("talker", options)
+{
+  // Create a publisher of "std_mgs/String" messages on the "chatter" topic.
+  pub_ = create_publisher<std_msgs::msg::String>("chatter", 10);
+
+  auto msg = std::make_unique<std_msgs::msg::String>();
+  msg->data = "Hello World";
+  RCLCPP_INFO(this->get_logger(), "Publishing: '%s'", msg->data.c_str());
+  std::flush(std::cout);
+
+  // Put the message into a queue to be processed by the middleware.
+  // This call is non-blocking.
+  pub_->publish(std::move(msg));
```

Comment:
@BMarchi if you check container output, you don't even have to publish.

Reply:
I can't see why I don't have to publish, can you explain? The container process is the one who gets the output

Reply:
I mean you can simply check the output the container generates on component load, no need for the component to generate extra output or even have any logic beyond these tests' purposes.